src/main/resources/templates/fragments/head.html

```diff
@@ -5,12 +5,6 @@
     name="viewport"
     content="width=device-width, initial-scale=1, shrink-to-fit=no"
   />
-  <meta id="_csrf" name="_csrf" th:content="${_csrf.token}" />
-  <meta
-    id="_csrf_header"
-    name="_csrf_header"
-    th:content="${_csrf.headerName}"
-  />
```

Comment:
I'm pretty sure removing this is breaking the Gatling Academy material!

Reply:
Really... I don't attend the last module of Gatling Academy.
Should we store the crsf in cookie only for this use case ?

Reply:
I’m pretty sure capture this token is part of the exercise. Check with Bastos and Thibaut.

Reply:
Checked with them. And indeed, there is a full video on this specific subject.
I will try to find how to manage that :+1:

Reply:
Found something useful ^^ PTAL
(see WebSecurityConfig.java)

Reply:
@Isammoc Have you tested that the Gatling tests in https://github.com/gatling/gatling-academy-module-2-java and https://github.com/gatling/gatling-academy-module-3 still work (beware: branches)?

Reply:
I didn't. But now I did.

All green (not tested every branches... about half)!